ansible/monitoring.yml

```diff
@@ -0,0 +1,42 @@
+- name: Configure monitoring services (DataDog)
+  hosts: all
+  remote_user: master
+  become: yes
+  tags: [datadog]
+
+  vars:
+    # separate var joined below just for readability
+    datadog_tags:
+      - "env:{{ datadog_env }}"
+      - "lsb:{{ ansible_lsb.description }}"
+      - "flavor:c{{ ansible_processor_vcpus }}.m{{ (ansible_memtotal_mb/1024)|round|int }}"
+    datadog_config:
+      apm_enabled: false
+      use_dogstatsd: yes
+      process_agent_enabled: true
+      hostname: "{{ ansible_fqdn }}"
+      tags: "{{ datadog_tags|join(',') }}"
+    datadog_checks:
```

Comment:
@dincho Where is the documentation of the reference of this `datadog_checks`? (I see multiple versions of the agent, and did not find clear pointer to documentation.)

Reply:
I fixed the DD role version to [1.4.0](https://github.com/aeternity/infrastructure/pull/7/files#diff-74a84d68b99baca3e0a906aba92bd4c9R2). So the docs should be [this](https://github.com/DataDog/ansible-datadog/blob/1.4.0/README.md), however this role just spit in a conf file that you provide as dictionary.

Agents configuration references/docs can be found [here](https://docs.datadoghq.com/integrations/)